coalib/parsing/ConfParser.py

```diff
@@ -123,6 +123,12 @@ def __parse_lines(self, lines, origin):
                 if key == '':
                     continue
 
+                if key in current_section.contents and keys != []:
```

Comment:
Nope, `current_keys` is not equal to `keys` all the time. This is the output that I receive when I remove `keys != []`
http://paste.ubuntu.com/24234922/